apps/nextjs-website/src/pages/404.tsx

```diff
@@ -0,0 +1,29 @@
+import { translations } from '@/_contents/translations';
+import SectionTitle from '@/components/molecules/SectionTitle/SectionTitle';
+import Layout, { LayoutProps } from '@/components/organisms/Layout/Layout';
+import { getProducts } from '@/lib/api';
+import { GetStaticProps, GetStaticPropsResult } from 'next';
+
+export type PageNotFoundProps = LayoutProps;
+
+export const getStaticProps: GetStaticProps<
+  PageNotFoundProps
+> = (): GetStaticPropsResult<LayoutProps> => {
+  return { props: { products: [...getProducts()] } };
+};
+
+export default function PageNotFound({ products }: PageNotFoundProps) {
+  const { pageNotFound } = translations;
```

Comment:
Non so se avrei fatto arrivare tutti i testi dalle traduzioni, ma in questo caso ci può anche stare. Forse non farei così le pagine "Privacy" e "Tos"

Reply:
I did not like having hard-coded parts in the code